.github/workflows/lint-test.yaml

```diff
@@ -39,6 +39,20 @@ jobs:
 
       - name: Run chart-testing (lint)
         run: ct lint --target-branch main
+      
+      - name: Run example check
+        working-directory: examples/opentelemetry-collector
+        run: |
+          for d in */ ; do
+            echo "Checking example: ${d}"
+            helm template example ../../charts/opentelemetry-collector --values "${d}values.yaml" --output-dir "./test/${d}"
+            if diff "${d}manifest" "./test/${d}opentelemetry-collector/templates" >/dev/null; then
+                echo "Passed ${d}";
+            else
+                echo "Failed ${d}. Check that manifest/ has been rendered with the latest ${d}values.yaml";
+                exit 1                
+            fi
```

Comment:
I think this should be replaced with a Makefile target. Which user can easily run before submitting a PR. Error message here should say to use that target as well

Reply:
Are you thinking each example would be a target that would render the manifest and flatten the result? Or a make file that would render each example?

Reply:
I think one make target should render all the examples

Reply:
Will do